salt/modules/kapacitor.py

```diff
@@ -117,6 +134,13 @@ def define_task(name,
     task_type
         Task type. Defaults to 'stream'
 
+    dbrps
+        A list of databases and retention policies in "dbname"."rpname" format
+        to fetch data from. For backward compatibility, the value of
+        'database' and 'retention_policy' will be merged as part of dbrps.
+
+        .. versionadded:: Fluorine
+
     database
         Which database to fetch data from. Defaults to None, which will use the
```

Comment:
@rallytime btw, I believe this has never been "really" true, it has been changed on [28.10.2015](https://github.com/influxdata/kapacitor/commit/19bf7e7b8b02fa8eafbe72c95f92645728dd14e2#diff-e58bf33a2f703ec10ac03af9342cd4f9R299) which means that happened even before the major change in v0.13.

Therefore, I will change this too.

Reply:
Ok sounds good.